dask_cuda/utils.py

```diff
@@ -1,6 +1,57 @@
 import toolz
 import os
+import math
+import numpy as np
+from multiprocessing import cpu_count
 from numba import cuda
+from pynvml import nvmlInit, nvmlDeviceGetHandleByIndex, nvmlDeviceGetCpuAffinity
+
+
+class CPUAffinity:
+    def __init__(self, cores):
+        self.cores = cores
+
+    def setup(self, worker=None):
+        print("before", os.sched_getaffinity(0))
+        os.sched_setaffinity(0, self.cores)
+        print("after ", os.sched_getaffinity(0))
+
+
+def bitmask_to_list(x, mask_bits=64):
```

Comment:
Can I ask for a small docstring here, perhaps with a small example?

Reply:
Renamed to `unpack_bitmask` and added docstring. I think the new name is more descriptive.